samples/boards/mec15xxevb_assy6853/src/main.c

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define LOG_LEVEL LOG_LEVEL_INF
+#include <logging/log.h>
+LOG_MODULE_DECLARE(mec15_brd_test);
+#include "power_mgmt.h"
+
+
+void main(void)
+{
+	test_pwr_mgmt_singlethread();
```

Comment:
test_pwr_mgmt_multithread ?

Reply:
Correct. Also added parameters to run same test with and without using logging.
Similarly there are hooks to keep track of entry/exit for each sleep state.